tests/core/util/test_streamable.py

```diff
@@ -315,7 +315,7 @@ class TestClassList(Streamable):
         a: List[uint8]
 
     # Does not have the required elements
-    with raises(AssertionError):
+    with raises(ValueError):
```

Comment:
it seems correct that an invalid length is a `ValueError` rather than an `AssertionError`.